util/unifyfs/src/unifyfs-rm.c

```diff
@@ -651,6 +821,50 @@ static int jsrun_terminate(unifyfs_resource_t* resource,
     return -errno;
 }
 
+/**
+ * @brief Launch data stage using IBM jsrun
+ *
+ * @param resource The job resource record
+ * @param args     The command-line options
+ *
+ * @return
+ */
```

Comment:
@craigsteffen I forgot to include a note in the comment, saying that "currently we are launching a process per node, but we could launch more processes based on the available cores per node in future."

Reply:
Which function should that comment go in?

Reply:
This has now been added to the new patch that will go in.